templates/projects/webhook.html

```diff
@@ -0,0 +1,73 @@
+{% extends "/base.html" %}
+{% set active_page = "projects" %}
+{% set active_project  = "all" %}
+{% import "projects/_helpers.html" as helper %}
+
+{% block content %}
+{% include "projects/js/webhook.js" %}
+<div class="row">
+    <div class="span3">
+        {{ helper.render_project_local_nav(project, 'webhook', current_user) }}
+    </div>
+    <div class="span9" style="min-height:400px">
+        {{ helper.render_project_title(project, upload_method) }}
+        <div style="text-align:center; padding-bottom:15px;">
+            <a id="webhooks-refresh" href="#" class="btn btn-info" style="display:none;" onclick="document.location.reload(true);"> Refresh</a>
+        </div>
+        <div class="table-responsive">
+            <table class="table table-hover">
+                <tr>
+                    <td><strong>{{_('Created')}}</strong></td>
+                    <td><strong>{{_('Updated')}}</strong></td>
+                    <td><strong>{{_('Payload')}}</strong></td>
+                    <td><strong>{{_('Response')}}</strong></td>
+                    <td><strong>{{_('Response Status Code')}}</strong></td>
+                    <td><strong>{{_('Repeat')}}</strong></td>
+                </tr>
+
+                {% for response in responses | sort(reverse=True, attribute='updated') %}
+                <tr id="webhook_tr_{{response.id}}">
+                    <td>
+                        {{ response.created }}
+                    </td>
+                    <td id="response_updated_{{response.id}}">{{ response.updated }}</td>
+                    <td>{{response.payload}}</td>
+                    <td id="response_{{response.id}}">{{response.response}}</td>
+                    <td id="response_status_code_{{response.id}}">{{response.response_status_code}}</td>
+                    <td>
+                        <button class="btn btn-info" onclick="run('{{project.short_name}}', {{response.id}})"><i class="icon-refresh"></i></button>
+                    </td>
+                </tr>
+                {% endfor %}
+            </table>
+        </div>
+    </div>
+</div>
+    <script>
+        var url = "../{{project.short_name}}/privatestream"; 
+        var source = new EventSource(url);
+        source.onmessage = function (event) {
+            if (alert.data != 1) {
+                var msg = JSON.parse(event.data);
+                if (msg.type == 'webhook') {
+                    var wh = msg.data;
+                    console.log(wh.updated);
+                    $("#response_updated_" + wh.id).text(wh.updated);
+                    $("#response_" + wh.id).text(wh.response);
+                    $("#response_status_code_" + wh.id).text(wh.response_status_code);
+                    if (wh.response_status_code == 200) {
```

Comment:
I would avoid comparing with the == operator, and use === instead as a general rule of thumb :+1: